content/ai/ai-tools.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: AI Tools
+description: |
+  Access to large language models and AI tools through CCV's computing infrastructure.
+---
+
+<ContentSection title="About ChatCCV Tools">
+CCV provides access to cutting-edge AI tools and large language models through our high-performance computing
+infrastructure. Whether you need to run LLMs for research, explore AI capabilities, or integrate AI into your
+computational workflows, we have the resources and expertise to support your needs.
+
+<Button
+  variant="primary_outlined"
+  size="md"
+  href="https://docs.ccv.brown.edu/ai-tools/privacy-statement"
+>
+  CCV AI Tools Privacy Statement
+</Button>
+
+  <CardGroup>
+    <StyledCard
+      iconName="FaFileAudio"
+      title="Transcribe"
+      footer={<Button size="md" href="https://ai.ccv.brown.edu/transcribe">Try Transcribe</Button>}>
```

Comment:
I think Isa wanted some different wording than "Try Transcribe", but I forgot what it was....